Quickstart guide fails with error `make: cross: No such file or directory` in WSL

The symptom:
```sh
$PACKAGE=hello_world make build
cross -v build --release --target aarch64-unknown-linux-gnu --package hello_world
make: cross: No such file or directory
make: *** [Makefile:225: target/aarch64-unknown-linux-gnu/release/hello_world] Error 127
```

The cause:
```sh
$ source init_env.sh 
Creating venv
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt install python3.10-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: /<REDACTED>/acap-rs/venv/bin/python3

-bash: venv/bin/activate: No such file or directory
```

The solution:
Installing `python3.10-venv`, removing `./venv/` if it exists and following the quickstart guide from the top again probably helps.
It is also possible to install cross without a virtual environment like `cargo install cross`, but circumventing the virtual environment is completely untested.

Actions:
* Document either this dependency or a workflow that does not need it?
* Clarify what has been tested on what systems?

--- a/README.md
+++ b/README.md
@@ -33,6 +33,7 @@ Ensure global prerequisites are installed:
 * Docker
 * Rust e.g. [using rustup](https://www.rust-lang.org/tools/install)
 * Python e.g. using [pyenv](https://github.com/pyenv/pyenv)
+  * When using the system python on some versions of Debian and Ubuntu, it is necessary to `apt install python3-venv`.
 
 Create, activate and populate the local development environment like
 

